22:46:10,077 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:10,114 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:10,124 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:10,132 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-21 to job manager
22:46:10,196 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:10,196 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:10,197 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:11,308 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:11,328 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:11,337 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:11,337 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,703 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (18/25)
22:46:46,706 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (18/25)
22:46:48,424 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (140/200)
22:46:48,424 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (140/200)
22:46:48,427 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (140/200)
22:46:48,428 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (140/200)
22:46:49,009 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (142/200)
22:46:49,009 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (142/200)
22:46:49,011 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (142/200)
22:46:49,013 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (142/200)
22:46:49,071 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (141/200)
22:46:49,071 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (141/200)
22:46:49,073 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (141/200)
22:46:49,075 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (141/200)
22:46:49,731 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (143/200)
22:46:49,731 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (143/200)
22:46:49,732 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (143/200)
22:46:49,733 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (143/200)
22:46:49,755 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (138/200)
22:46:49,755 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (138/200)
22:46:49,757 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (138/200)
22:46:49,758 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (138/200)
22:46:50,450 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (144/200)
22:46:50,450 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (144/200)
22:46:50,451 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (144/200)
22:46:50,452 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (144/200)
22:46:50,991 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (139/200)
22:46:50,992 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (139/200)
22:46:50,994 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (139/200)
22:46:50,995 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (139/200)
22:46:53,174 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (137/200)
22:46:53,174 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (137/200)
22:46:53,177 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (137/200)
22:46:53,178 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (137/200)
22:48:24,862 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (18/25)
22:48:24,863 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (18/25)
22:49:27,096 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-2491093647324796168_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34733 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:27,097 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2491093647324796168_1722 bad datanode[0] 130.149.21.25:45020
22:49:27,098 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2491093647324796168_1722 in pipeline 130.149.21.25:45020, 130.149.21.24:45020, 130.149.21.3:45020: bad datanode 130.149.21.25:45020
22:49:30,646 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5353198248872470430_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34735 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:30,647 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5353198248872470430_1722 bad datanode[0] 130.149.21.25:45020
22:49:30,648 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5353198248872470430_1722 in pipeline 130.149.21.25:45020, 130.149.21.23:45020, 130.149.21.21:45020: bad datanode 130.149.21.25:45020
22:49:36,419 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-2329347928451241819_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34726 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:36,420 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2329347928451241819_1722 bad datanode[0] 130.149.21.25:45020
22:49:36,420 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2329347928451241819_1722 in pipeline 130.149.21.25:45020, 130.149.21.33:45020, 130.149.21.63:45020: bad datanode 130.149.21.25:45020
22:49:36,614 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_7156642511542281923_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34722 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:36,615 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7156642511542281923_1722 bad datanode[0] 130.149.21.25:45020
22:49:36,616 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7156642511542281923_1722 in pipeline 130.149.21.25:45020, 130.149.21.61:45020, 130.149.21.26:45020: bad datanode 130.149.21.25:45020
22:49:43,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1932957235215987491_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34728 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:43,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1932957235215987491_1722 bad datanode[0] 130.149.21.25:45020
22:49:43,606 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1932957235215987491_1722 in pipeline 130.149.21.25:45020, 130.149.21.26:45020, 130.149.21.29:45020: bad datanode 130.149.21.25:45020
22:49:53,229 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8119914110457455867_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34730 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:53,231 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8119914110457455867_1722 bad datanode[0] 130.149.21.25:45020
22:49:53,231 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8119914110457455867_1722 in pipeline 130.149.21.25:45020, 130.149.21.29:45020, 130.149.21.64:45020: bad datanode 130.149.21.25:45020
22:50:06,472 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7279396993338870453_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34739 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:06,473 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7279396993338870453_1722 bad datanode[0] 130.149.21.25:45020
22:50:06,473 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7279396993338870453_1722 in pipeline 130.149.21.25:45020, 130.149.21.22:45020, 130.149.21.23:45020: bad datanode 130.149.21.25:45020
22:50:15,937 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8740599349916051659_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:34742 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:15,938 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8740599349916051659_1722 bad datanode[0] 130.149.21.25:45020
22:50:15,939 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8740599349916051659_1722 in pipeline 130.149.21.25:45020, 130.149.21.26:45020, 130.149.21.60:45020: bad datanode 130.149.21.25:45020
22:50:33,224 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-2491093647324796168_1748java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:43421 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:33,225 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2491093647324796168_1748 bad datanode[0] 130.149.21.24:45020
22:50:33,225 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2491093647324796168_1748 in pipeline 130.149.21.24:45020, 130.149.21.3:45020: bad datanode 130.149.21.24:45020
22:50:36,748 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5353198248872470430_1771java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:37798 remote=/130.149.21.23:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:36,749 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5353198248872470430_1771 bad datanode[0] 130.149.21.23:45020
22:50:36,749 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5353198248872470430_1771 in pipeline 130.149.21.23:45020, 130.149.21.21:45020: bad datanode 130.149.21.23:45020
22:50:42,520 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-2329347928451241819_1783java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:44675 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:42,521 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2329347928451241819_1783 bad datanode[0] 130.149.21.33:45020
22:50:42,521 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-2329347928451241819_1783 in pipeline 130.149.21.33:45020, 130.149.21.63:45020: bad datanode 130.149.21.33:45020
22:50:42,714 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_7156642511542281923_1784java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:45213 remote=/130.149.21.61:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:42,715 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7156642511542281923_1784 bad datanode[0] 130.149.21.61:45020
22:50:42,715 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7156642511542281923_1784 in pipeline 130.149.21.61:45020, 130.149.21.26:45020: bad datanode 130.149.21.61:45020
22:50:49,703 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1932957235215987491_1794java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:57051 remote=/130.149.21.26:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:49,704 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1932957235215987491_1794 bad datanode[0] 130.149.21.26:45020
22:50:49,705 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1932957235215987491_1794 in pipeline 130.149.21.26:45020, 130.149.21.29:45020: bad datanode 130.149.21.26:45020
22:50:59,328 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8119914110457455867_1813java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:41810 remote=/130.149.21.29:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:59,328 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8119914110457455867_1813 bad datanode[0] 130.149.21.29:45020
22:50:59,328 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8119914110457455867_1813 in pipeline 130.149.21.29:45020, 130.149.21.64:45020: bad datanode 130.149.21.29:45020
22:51:12,570 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7279396993338870453_1838java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:59040 remote=/130.149.21.22:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:12,572 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7279396993338870453_1838 bad datanode[0] 130.149.21.22:45020
22:51:12,572 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7279396993338870453_1838 in pipeline 130.149.21.22:45020, 130.149.21.23:45020: bad datanode 130.149.21.22:45020
22:51:22,055 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8740599349916051659_1872java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.25:57068 remote=/130.149.21.26:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:22,056 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8740599349916051659_1872 bad datanode[0] 130.149.21.26:45020
22:51:22,057 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8740599349916051659_1872 in pipeline 130.149.21.26:45020, 130.149.21.60:45020: bad datanode 130.149.21.26:45020
